app/vmock.go

```diff
@@ -67,7 +83,7 @@ func wireValidatorMock(conf Config, pubshares []eth2p0.BLSPubKey, sched core.Sch
 			})
 		}
 
-		onStartup = false
+		setOnStartup()
```

Comment:
nit, rename to `clearOnStartup` or `startedUp`. `setOnStartup` sounds like you are setting "onStartup==true"